### DISCUSSION TITLE
text color when using as="a"

### DISCUSSION THREAD

**Discussion by Schmell** [chakra-ui/chakra-ui#6022] [Q&A]
I have a button I use to navigate with an as="a" attribute and I am using colorScheme=blue, how ever the text color is now black instead of white.
I think its because I have visited said link, but I don't know how to make it permanently white.

**Comment by aroyan** [comment]
Try adding pseudo selector as props on a component that you used. For example in this case you said you use Button so it will be like this 
```javascript
    <Button _visited={{color: "white"}}>Contact Me</Button>
```

or if you have something like this
```javascript
    <Text as='a' _visited={{color: "white"}}>Chakra UI</Text>
```

Further information you can check out [Props](https://chakra-ui.com/docs/styled-system/features/style-props#pseudo) documentation by ChakraUI and [:visited](https://developer.mozilla.org/en-US/docs/Web/CSS/:visited) documentation by MDN Docs

**Reply by Schmell** [reply]
Thanks @aroyan I did not realize there were pseudo attributes I can use. That's awesome! 👍 
This works for text in the button but not the icon
```javascript
<Button
    as="a"
    href="/upload"
    colorScheme="blue"
    variant="outline"
    boxShadow="md"
    leftIcon={<TriangleUpIcon />}
    _visited={{ color: "blue" }}
    >
    Upload
</Button>
```

**Reply by aroyan** [reply]
> Thanks @aroyan I did not realize there were pseudo attributes I can use. That's awesome! 👍 This works for text in the button but not the icon
> 
> ```js
> <Button
>     as="a"
>     href="/upload"
>     colorScheme="blue"
>     variant="outline"
>     boxShadow="md"
>     leftIcon={<TriangleUpIcon />}
>     _visited={{ color: "blue" }}
>     >
>     Upload
> </Button>
> ```

The default behavior is shouldn't have visited selector as far as I know, can you provide a codesandbox for this issue?
